packages/smooth_app/lib/pages/image_crop_page.dart

```diff
@@ -6,137 +6,153 @@ import 'package:image_cropper/image_cropper.dart';
 import 'package:image_picker/image_picker.dart';
 import 'package:smooth_app/generic_lib/dialogs/smooth_alert_dialog.dart';
 
+/// Returns the file path of an image after it's been cropped.
+///
+/// This is the "old" problematic version; to be rapidly changed.
+Future<String?> getCroppedPath(
+  final BuildContext context,
+  final String inputPath,
+) async =>
+    (await ImageCropper().cropImage(
+      sourcePath: inputPath,
+      aspectRatioPresets: <CropAspectRatioPreset>[
+        CropAspectRatioPreset.square,
+        CropAspectRatioPreset.ratio3x2,
+        CropAspectRatioPreset.original,
+        CropAspectRatioPreset.ratio4x3,
+        CropAspectRatioPreset.ratio16x9
+      ],
+      uiSettings: <PlatformUiSettings>[
+        AndroidUiSettings(
```

Comment:
There is no special config required for iOS?

Reply:
I didn't change that.
There's no "required" config for Android either, it was just the way it was coded before I refactored.
That's interesting, because iOS config includes "initial crop" parameters (Android does not). Will be fixed with the new version of the crop tool.